README.md

```diff
@@ -0,0 +1,30 @@
+# IDR systems documentation
+
+[Image Data Resource (IDR)](https://idr-demo.openmicroscopy.org/) is an online, public data repository that seeks to store, integrate and serve image datasets from published scientific studies.
+The IDR is also a platform that is entirely built with open-source components and tools, and these documents describe how to build and manage your own version of the IDR.
+
+The IDR is currently hosted on OpenStack at the [EBI](http://www.ebi.ac.uk/).
```

Comment:
I know but I am not convinced that when you used the abbreviated name. ``the`` is suitable